_frc/4900/4941.md

```diff
@@ -9,6 +9,23 @@ team:
   sponsors:
     - Howard High School
   links:
-    Website: 
+    Code: https://github.com/robobibb/
+    Twitter(@FRC4941): https://twitter.com/FRC4941
+    Facebook: https://fb.com/teamrobobibb/
+robot_code:
+  2014 - 2015:
```

Comment:
Would having different names for the parts of the code be allowed as is done here?
```
  2017:
  - Main-Robot:
    - https://github.com/robobibb/main-bot-2017
    - C++
  - Practice-Robot:
    - https://github.com/robobibb/test-bot-2017
    - C++
```